reference/auth.v1.yaml

```diff
@@ -722,6 +722,27 @@ paths:
       security:
         - serverToken: []
 
+  '/auth/v1/users/{userId}/device_tokens':
+    parameters:
+      - $ref: './common/parameters/tidepooluserid.yaml'
+    post:
+      operationId: PostDeviceToken
+      summary: Post Device Token
+      description: >-
+        Stores a token used to send notifications to a device such as a mobile phone.
+      responses:
+        '200':
+          description: 200 OK
+        '400':
+          $ref: './common/responses/badrequest.v1.yaml'
+        '403':
+          $ref: './common/responses/forbidden.v1.yaml'
+      tags:
+        - Authentication
+        - Internal
```

Comment:
I thought `Internal` meant "backend only". Maybe I'm mistaken.

Reply:
> I thought `Internal` meant "backend only". Maybe I'm mistaken.

The only thing I have to go on is this:

https://github.com/tidepool-org/TidepoolApi/blob/eric%2Fdevice-tokens/reference/auth.v1.yaml#L33-L34

The endpoint is only intended for use by the Tidepool Care Partner App. So not backend, but Tidepool for sure. My interpretation was that that constituted internal use by Tidepool.

I can take it or leave it, but to me, this tag is to warn non-Tidepoolers, that this endpoint might change more or less without notice, and that unless they really know what they're doing, they shouldn't be using it. Those things feel correct here to me. Maybe it's meant to warn away non-backed uses though? I dunno. 

Is there a better tag to use in its place? Something like "Tidepool-only"?

Reply:
The description of the Authentication tag is:

>     description: >-
>      Login and logout.

The device tokens aren't used for either of those, so I guess I'll remove it too.